Mathlib/Order/Defs/Unbundled.lean

```diff
@@ -57,6 +57,12 @@ instance {α : Sort*} {r : α → α → Prop} [IsTrans α r] : Trans r r r :=
 instance (priority := 100) {α : Sort*} {r : α → α → Prop} [Trans r r r] : IsTrans α r :=
   ⟨fun _ _ _ => Trans.trans⟩
 
+/-- `IsCotrans X r` means the binary relation `r` on `X` is cotransitive. A cotransitive
```

Comment:
Are the changes in this file necessary? If not, please split out

Reply:
Oops, this was from an old way of talking about them that we got rid of. Right. Yes, don't need these